Core/Src/sensors/APPS.c

```diff
@@ -16,15 +16,14 @@
 #include "util/util.h"
 
 typedef struct {
-    uint16_t value[APPS_MATRIX_LENGTH];
-    float fix_mul[APPS_MATRIX_LENGTH];
-    float fix_add[APPS_MATRIX_LENGTH];
+    uint16_t deadzone_limits[APPS_MATRIX_LENGTH];
+    float adjust_parameters[APPS_MATRIX_LENGTH];
```

Comment:
Nessa struct acho melhor não usar um vetor, e sim uma variavel pra cada. Vai ajudar na legibilidade. cpa rola até de colocar o max e min aqui tbm

```suggestion
    uint16_t deadzone_lower_limit;
    uint16_t deadzone_upper_limit;
    float adjust_parameters_slope;
    float adjust_parameters_intercept;
```